### PR TITLE
Add docs to all plugins

### PR DESCRIPTION
**Pull request by jorana**
**What is the problem / what does the code in this PR do**
Since #185, #187 and #189 we are now uploading the nT data-structure to the documentation. With this PR @WenzDaniel and I are adding brief descriptions to the plugins to contribute to the documentation.

### CODE REVIEW

File: straxen/plugins/peaklet_processing.py [hunk 1]
```diff
@@ -53,6 +53,20 @@
...
 )
 class Peaklets(strax.Plugin):
+    """
+    Split records into:
+        -peaklets
+        -lone_hits
+
+    Peaklets are very aggressively split peaks such that we are able to find S1-S2s even
+        if they are close to each other. (S2) Peaks that are split into too many peaklets
+        will be merged later on.
```
Comment (WenzDaniel) [inline straxen/plugins/peaklet_processing.py:63]: Maybe also add one line for lone_hits?

File: straxen/plugins/pulse_processing.py [hunk 2]
```diff
@@ -90,8 +90,9 @@
 class PulseProcessing(strax.Plugin):
     """
     1. Split raw_records into:
-     - tpc_records
+     - (tpc) records
      - aqmon_records
+     - pulse_counts
```
Comment (WenzDaniel) [inline straxen/plugins/pulse_processing.py:95]: I think also for pulse_counts we should write 1-2 sentences.

**Review by WenzDaniel — APPROVED** [review]

File: straxen/plugins/peaklet_processing.py (hunk 1 above)
Comment (WenzDaniel) [reply straxen/plugins/peaklet_processing.py:63]: Added something

File: straxen/plugins/pulse_processing.py (hunk 2 above)
Comment (WenzDaniel) [reply straxen/plugins/pulse_processing.py:95]: Done

**Comment by WenzDaniel** [comment]
I think I added all the missing parts. Unfortunately I was not able to get graphviz running to check whether the format is correct.

**Comment by jorana** [comment]
Hi Daniel! Thanks for the help! I just went over all of them a last time and noticed we had quite a different style (in line-breaks). I tried smoothing it a little by abiding the PEP8 convention on docstrings https://www.python.org/dev/peps/pep-0008/#maximum-line-length (72 characters).

**Comment by WenzDaniel** [comment]
Hi, I thought I would use this convention already (at least for the nveto part). At least I thought to remember that I set-up guiding line in PyCharm which follows this rule. The other docs I wrote "free-hand" directly in github.

**Comment by jorana** [comment]
> 
> 
> Hi, I thought I would use this convention already (at least for the nveto part). At least I thought to remember that I set-up guiding line in PyCharm which follows this rule. The other docs I wrote "free-hand" directly in github.

Therefore I also didn't alter things for the nveto part (in d2662b2) but more on the descriptions I wrote earlier.

**Pull request merged**